grafana-plugin/e2e-tests/schedules/addOverride.test.ts

```diff
@@ -20,3 +20,37 @@ test('default dates in override creation modal are correct', async ({ adminRoleP
   expect(overrideFormDateInputs.start.isSame(expectedStart)).toBe(true);
   expect(overrideFormDateInputs.end.isSame(expectedEnd)).toBe(true);
 });
+
+test('Fills in override time and reacts to timezone change', async ({ adminRolePage }) => {
+  const { page, userName } = adminRolePage;
+
+  const onCallScheduleName = generateRandomValue();
+  await createOnCallSchedule(page, onCallScheduleName, userName, false);
+
+  await clickButton({ page, buttonText: 'Add override' });
+
+  const overrideStartEl = page.getByTestId('override-start');
+  await changeDatePickerTime(overrideStartEl, '02');
+  await expect(overrideStartEl.getByTestId('date-time-picker').getByRole('textbox')).toHaveValue('02:00');
+
+  const overrideEndEl = page.getByTestId('override-end');
+  await changeDatePickerTime(overrideEndEl, '12');
+  await expect(overrideEndEl.getByTestId('date-time-picker').getByRole('textbox')).toHaveValue('12:00');
+
+  await page.getByRole('dialog').click(); // clear focus
+
+  await page.getByTestId('timezone-select').locator('svg').click();
+  await page.getByTestId('timezone-select').getByText('GMT', { exact: true }).click();
+
+  // expect times to go back by -3
+  await expect(overrideStartEl.getByTestId('date-time-picker').getByRole('textbox')).toHaveValue('23:00');
+  await expect(overrideEndEl.getByTestId('date-time-picker').getByRole('textbox')).toHaveValue('09:00');
```

Comment:
On CI this test can run in different timezones so you can't really predict how many hours will be added / subtracted. I think to make it reliable the test can include the following steps:
1. Set GMT in timezone select
2. Add override
3. Change offset in timezone select (let's say to GMT-3)
4. Assert times in date time picker

Reply:
I thought by default its set to GMT+3, I'll just call the fuction that sets it to Moscow time then as in other places. It changes back to GMT so there's a 3 hour difference, initially I wanted to fill in a value like Hawaii to have a larger gap, but I couldnt figure out how to instruct playwright do it as the component is being rendered in a portal